examples/config/lua-rocks-config.lua

```diff
@@ -0,0 +1,9 @@
+local neorocks = require("plenary.neorocks")
+
+if(neorocks._is_setup)
+then
+  neorocks.ensure_installed('rapidjson', 'rapidjson')
+else
+  neorocks.setup {}
+end
```

Comment:
What's up with this? Maybe this is the reason why you need to restart it multiple times?

Reply:
yeah... so if i didn't do it like this it would blow up something to do with cmake... i'm kinda on the fence about just figuring how to jam `luajitPackages.<xyz>` from nix into the rtp